Documentation/dynamic-rupture.rst

```diff
@@ -153,3 +153,37 @@ Linear-Slip Weakening Friction
 
 Rate-and-State Friction
 ^^^^^^^^^^^^^^^^^^^^^^^
+
+Thermal Pressurization
+~~~~~~~~~~~~~~~~~~~~~~
+
+Following Noda & Lapusta (2010) Seissol can now account for thermal pressurization of pore fluids mimicking the effects of rapid co-seismic slip generating heat that increases temperature and pore pressure in case of low hydraulic diffusivity of the surrounding rock. Consequently, the elevated pore pressure reduces the effective normal stress, causing dynamic weakening. 
+By default, thermal pressurization is inactive. You can activate thermal pressurization in the following parameter file:
+
+.. code-block:: Fortran
+
+  &DynamicRupture
+  thermalPress = 1                  ! 0: inactive; 1: active
+  IniTemp = 483.15                  ! [K]
+  IniPressure = -80.0e6             ! have to be added to normal stress in your initial stress yaml file [Pa]
```

Comment:
Negative pressure seems strange to me.

Reply:
because of sign convention in SeisSol. Therefore I explicitly emphasize here that the initial pore pressure needs to be negative as well